nomad/structs/structs.go

```diff
@@ -73,6 +73,7 @@ const (
 	ACLTokenUpsertRequestType
 	ACLTokenDeleteRequestType
 	ACLTokenBootstrapRequestType
+	ReasonWithinPolicy = "Restart within policy"
```

Comment:
This seems out of place... At least put a space between it and the "enum" above + a comment describing its intended purpose.